pages/common/surfraw.md

```diff
@@ -0,0 +1,25 @@
+# surfraw
+
+> Surfraw is a CLI to query a variety of web search engines.
+> It consists of a collection of elvi, each of which knows how to search a specific website.
+> More information: <http://surfraw.org/>.
+
+- Show the list of elvi:
```

Comment:
The doc says `To see the list of elvi type`. The plural and singular elvi, elvis, type in the doc is strange to me.
Based on https://gitlab.com/surfraw/Surfraw/-/wikis/current-elvi we can make this better to define `elvi`:
```suggestion
- Display the list of supported website search scripts (elvi):
```